CMSIS/NN/Include/arm_nnsupportfunctions.h

```diff
@@ -287,6 +287,40 @@ arm_status arm_nn_mat_mul_core_4x_s8(const int32_t row_elements,
                                      int32_t *const sum_col,
                                      int32_t *const output);
 
+/**
+ * @brief s8 Vector by Matrix (transposed) multiplication
+ *
+ * @param[in]      lhs             Input left-hand side vector
+ * @param[in]      rhs             Input right-hand side matrix (transposed)
+ * @param[in]      bias            Input bias
+ * @param[out]     dst             Output vector
+ * @param[in]      lhs_offset      Offset to be added to the input values of the left-hand side vector
+ * @param[in]      rhs_offset      Offset to be added to the input values of the right-hand side matrix
+ * @param[in]      dst_offset      Offset to be added to the output values
+ * @param[in]      dst_multiplier  Output multiplier
+ * @param[in]      dst_shift       Output shift
+ * @param[in]      k               Number of elements in the left-hand side input vector
```

Comment:
could you rename k and n as len_/rhs to make it easier to follow in the code?

Reply:
Hi @felix-johnny , "k" and "n" are used to follow the convention used by blas to define the number of inner accumulations and output columns (http://www.netlib.org/lapack/explore-html/db/dc9/group__single__blas__level3_gafe51bacb54592ff5de056acabd83c260.html#gafe51bacb54592ff5de056acabd83c260). We use this convention also in other ML libraries as well. Do you prefer to change or keep those names?